### DISCUSSION TITLE
AWS VPC and CIDR

### DISCUSSION THREAD

**Discussion by maheshkoheda** [learndevopsonline/learndevopsonline#1379] [Interview Questions]
When to use 10.0.0.0/16? what is the significance of 16? Can we use any other number in place of 16 here?

When to use 10.0.0.0/24?what is the significance of 24? Can we use any other number in place of 24 here?

**Comment by sreedharm07** [comment]
if you want max num of ips give /16. i think this attached image will helps you to when you use which number

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/caa27fca-55bb-4d61-9a10-e5ef835b8336)
